### PR TITLE
travis: update linux version & add python virtualenv

### PR DESCRIPTION
**Pull request by lvaccaro**
Following https://github.com/ElementsProject/lightning/issues/3779 , need to use python >= 3.6 available from travis linux machine bionic.
Add python3 environment to install lightning dependency.

### CODE REVIEW

**Comment by lvaccaro** [comment]
> Suggest to change the clang-format config to use  [more conventional settings for the C language](https://github.com/ElementsProject/lightning/blob/master/doc/STYLE.md#care-and-feeding-of-your-fellow-coders) ?
From @darosior

File: .travis.yml [hunk 1]
```diff
@@ -16,10 +16,14 @@ script:
   - git clone https://github.com/ElementsProject/lightning.git
   - cd lightning
   - git checkout v0.9.0-1
-  - pip3 install -r requirements.txt
+  - python3 -m virtualenv venv
+  - source venv/bin/activate
+  - pip install -r requirements.txt
+  - pip install mrkd
```
Comment (darosior) [inline .travis.yml:22]: nit: `mrkd` should be part of the `requirements.txt` ? If it is not we should report it upstream

File: .clang-format [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+Language:        Cpp
+BasedOnStyle: LLVM
+IndentWidth: 8
+UseTab: Always
+BreakBeforeBraces: Linux
```
Comment (darosior) [inline .clang-format:6]: :heart:

**Review by darosior — APPROVED** [review]
ACK 6528d0d8c446a6774459101cd00d4d924c0a15d3

Thank you !

File: .travis.yml (hunk 1 above)
Comment (lvaccaro) [reply .travis.yml:22]: right, thanks

**Pull request merged**